README.md

```diff
@@ -267,11 +267,11 @@ var logDirectory = path.join(__dirname, 'log')
 fs.existsSync(logDirectory) || fs.mkdirSync(logDirectory)
 
 // create a rotating write stream
-var accessLogStream = FileStreamRotator.getStream({
-  date_format: 'YYYYMMDD',
-  filename: path.join(logDirectory, 'access-%DATE%.log'),
-  frequency: 'daily',
-  verbose: false
+var accessLogStream = rotatingFileStream('access.log', {
```

Comment:
So this is no longer doing the exact same thing the old example used to do? Is it possible to implement the same rotation example as the previous one? Or perhaps update the description of the example to reflect the new behavior of the example?